W1-M11-LINQ/Program.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+namespace LinqFaroShuffle
+
+{
+
+    class Program {
+        static void Main(string[] args)
+        {
+            /// <summary> 

```

Comment:
You cannot use XML Comments within a method block.

Here it is within static void Main.

Inside, use // for comments or /* */ for multi line comments